src/xenia/gpu/trace_player.h

```diff
@@ -49,6 +56,10 @@ class TracePlayer : public TraceReader {
   GraphicsSystem* graphics_system_;
   int current_frame_index_;
   int current_command_index_;
+  std::atomic<bool> playing_trace_ = false;
```

Comment:
atomic here isn't doing what's intended - better to drop it so it's not confusing